### DISCUSSION TITLE
where is this file? src/redo/airway.mk.  I don't see a redo folder

### DISCUSSION THREAD

**Discussion by njbowen** [biostars/biostar-handbook-issues#327] [Q&A] [answered]
make -f src/redo/airway.mk design run
from this link:
https://www.biostarhandbook.com/books/workflows/workflows/airway-rnaseq/?h=

**Comment by ialbert** [comment, marked as answer]
it is located in the workflows  directory I will correct that link

    ./src/workflows/airway.mk

**Reply by njbowen** [reply]
Yes, I've been altering the workflows/airway.mk
for my projects.  

The typo is on the page near the bottom: 
The complete workflow

The complete workflow can be run in one shot with:


make -f src/redo/airway.mk design run


It might be helpful in the design.csv explanation part further up the page of the workflow to give the 4 column curated design.csv file to the user.
